File: CAPITULO 01/Java/ex18.java
import java.util.*;
import java.io.*;
public class ex18{
	public static void main(String[]args){
		int saco,gato1,gato2,kg_saco,peso_saco,fim_saco;
		float sobra_saco;
		Scanner dado = new Scanner(System.in);
		System.out.println("Digite o peso do saco de ração em kg: ");
		saco = dado.nextInt();
		kg_saco = 1000;
		peso_saco = saco * kg_saco;
		System.out.println("Digite a quantidade de ração fornecida para o gato 1 (em gramas): ");
		gato1 = dado.nextInt();
		System.out.println("Digite a quantidade de ração fornecida para o gato 2 (em gramas): ");
		gato2 = dado.nextInt();
		sobra_saco =(peso_saco - (5*(gato1 + gato2)));
		System.out.println(" "+sobra_saco+" gramas e o que sobrara após 5 dias");
	}
}

File: CAPITULO 01/C++/ex18.c
#include <stdio.h>
int main(){
    int saco,gato1,gato2,consumo,peso_saco;
    printf("Digite o peso do saco em (kg): " );
    scanf("%d%*c",&saco);
    printf("Digite a quantidade consumida pelo gato 1 (em gramas): ");
    scanf("%d%*c",&gato1);
    printf("Digite a quantidade consumida pelo gato 2 (em gramas): ");
    scanf("%d%*c",&gato2);
    peso_saco = saco * 1000;
    consumo = peso_saco - (5*(gato1+gato2));
    printf("%.3d\n" ,consumo);
    system("pause");
    return 0;
}
